samza-core/src/main/java/org/apache/samza/config/SystemConfig.java

```diff
@@ -96,6 +96,23 @@ public SystemConfig(Config config) {
               .getAdmin(systemNameToFactoryEntry.getKey(), this)));
   }
 
+  /**
+   * Get {@link SystemAdmin} instances for all the systems defined in this config and set with an adminLabel.
+   * This function provides an extra input parameter to {@link #getSystemAdmins}, which can be used to provide extra
+   * information for the admin instance, e.g. ownership of client instance, to help better identify admins in logs,
+   * threads and client instances etc., along with other relevant information like systemName.
+   *
+   * @param adminLabel a string to provide info the admin instance.
+   * @return map of system name to {@link SystemAdmin}
+   */
+  public Map<String, SystemAdmin> getSystemAdmins(String adminLabel) {
```

Comment:
can we change implementation of getSystemAdmins() to {
  return getSystemAdmins("").
}

Reply:
This file is updated. The version you reviewed is the outdated one. Please see: https://github.com/apache/samza/pull/1420/files#diff-2063da999ce81031b6015c31991f60c4R100